### ISSUE TITLE
[Issue]: Incompatible with firewalld version 1.1.0

### ISSUE THREAD

**Issue by lopopylopy** [jellyfin/jellyfin#8587]
### Please describe your bug

In Fedora Linux, when trying to use the command `sudo dnf system-upgrade download --releasever=37` to upgrade to Fedora 37, a transaction error occurs.

> Error: Transaction test error: file /usr/lib/firewalld/services/jellyfin.xml from install of firewalld-1.2.1-1.fc37.n

The new version of firewalld in Fedora 37 "now ships a service definition file ... which is in conflict with the one shipped by Jellyfin" ([chrisawi](https://www.reddit.com/r/Fedora/comments/y89ttc/comment/isznvo8/?utm_source=share&utm_medium=web2x&context=3))

This prevents `system-upgrade` to actually upgrade the system unless the user is to permanently delete the Jellyfin package. Fedora 37 is still in beta, but this is potentially critical when it releases on October 25.

### Jellyfin Version

Other

### if other:

10.8.5

### Environment

```markdown
- OS: Fedora
- Virtualization:
- Clients:
- Browser:
- FFmpeg Version:
- Playback Method:
- Hardware Acceleration:
- Plugins:
- Reverse Proxy:
- Base URL:
- Networking:
- Storage:
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by yodatak** [comment]
Same for me

**Comment by yodatak** [comment]
We should also bump all the reference of fedora 36 to 37 in [deployment/Dockerfile.fedora.amd64](https://github.com/jellyfin/jellyfin/blob/b1e0b4ee8940a7a38c766dff7a762d6e29b31412/deployment/Dockerfile.fedora.amd64)

and https://github.com/jellyfin/jellyfin/blob/c7edf8b565e7d52400cf65e747a8e800ecd3a036/fedora/jellyfin.spec

**Comment by yodatak** [comment]
I cannot install after the upgrade 
Erreur : Erreur de la transaction de test :
  le fichier /usr/lib/firewalld/services/jellyfin.xml de l'installation de jellyfin-server-10.8.5-1.fc36.x86_64 entre en conflit avec le fichier du paquet firewalld-1.2.1-1.fc37.noarch

**Comment by yodatak** [comment]
I try to make a fix here but as not tested yet https://github.com/jellyfin/jellyfin/pull/8644

**Comment by yodatak** [comment]
But for now it don't work and the platform are not the same in jellyfin and jellyfin-web, and the version in fedora/jellyfin.spec and https://github.com/jellyfin/jellyfin-web/blob/master/fedora/jellyfin-web.spec does not match so we have to bump the version in jellyfin-web also

**Comment by anthonylavado** [comment]
Incoming fix with #8646 8646

**Comment by tebrevik** [comment]
Easy fix. Uninstall firewalld and use something else...

Firewalld should not ship application specific definition files which creates too strong bindings. What if Jellyfin wants to add new ports in a release, should that have to wait for a firewalld release?  In my strong opinion this is a firewalld bug.
